webapp/src/components/settings/WakuForm.tsx

```diff
@@ -27,6 +40,32 @@ function WakuForm() {
 		};
 	}, [waku]);
 
+	// Waku does not support setting alternative harbour deployments, so read
+	// the harbour address and chain from the current settings and decide
+	// whether or not Waku is supported.
+	useEffect(() => {
+		let cancelled = false;
+		getHarbourChainId({ rpcUrl }).then((chainId) => {
+			if (cancelled) {
+				return;
+			}
+
+			const customHarbour =
+				harbourAddress && harbourAddress !== HARBOUR_ADDRESS;
```

Comment:
theoretically you don't need the `harbourAddress && `

Reply:
We do, as `harbourAddress` can be undefined (in which case, we don't have a custom harbour).